File: UVa/cpe_study/week6/11150/sol_1.cpp
#include <bits/stdc++.h>
#define endl "\n"
#define IO ios::sync_with_stdio(0), cin.tie(0), cout.tie(0)
#define MAXN maxn
#define MOD modn

using namespace std;

signed main(){
    int n;
    while(cin>>n) {
        int sum = 0, emp = 0;
        while(n+emp > 2) {
            sum += n; emp += n;
            n = emp / 3;
            emp %= 3;
        }
        sum += n;
        if(n+emp == 2) sum++;
        cout<<sum<<endl;
    }
    return 0;
}

File: UVa/cpe_study/week6/11150/sol_1.py
try:
    while True:
        n = int(input())
        res = emp = 0
        while n+emp>2:
            res+=n
            emp+=n
            n=emp//3
            emp%=3
        res+=n
        if n+emp==2:
            res+=1
        print(res)

except:
    pass